src/django/pfb_analysis/countries.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings
+import pycountry
+
+
+def get_country_config(alpha_2):
+    return settings.COUNTRY_CONFIG.get(alpha_2, settings.COUNTRY_CONFIG['default'])
+
+
+def use_subdivisions(alpha_2):
+    return settings.COUNTRY_CONFIG.get(alpha_2, settings.COUNTRY_CONFIG['default'])['subdivisions']
+
+
+def subdivisions_for_country(alpha_2):
+    """ Subdivisions for the given country, if we want to track them
+
+    Returns a list of subdivisions, as {name, code, type}, for the given country, but
+    only if the country config in settings says to track subdivisions for the country. Otherwise
+    returns None.
+    """
+    if use_subdivisions(alpha_2):
+        return sorted(
+            [{'name': s.name, 'code': s.code[3:], 'type': s.type} for s in
```

Comment:
What is in the field getting sliced for the code? Could `code` ever be empty or too short for the slice?

Reply:
Oops, should have commented that.  The subdivision codes all start with the alpha_2 country code and a hyphen.  We could use the whole thing and strip the prefix for presentation, but I don't think having the prefixes is likely to be useful, so it seems simpler to strip them off up front.